backend/validators/social_media_validator.py

```
@@ -0,0 +1,816 @@
+                    
+        except Exception as e:
+            logger.error(f"Token refresh error for {platform}: {e}")
+            return {"success": False, "reason": str(e)}
+    
+    async def _refresh_facebook_token(self, refresh_token: str) -> Dict:
+        """Refresh Facebook access token"""
+        try:
+            url = "https://graph.facebook.com/v18.0/oauth/access_token"
+            params = {
+                "grant_type": "fb_exchange_token",
+                "client_id": os.getenv("FACEBOOK_APP_ID"),
```

Ruff (0.12.2)

Undefined name `os`

(F821)